src/plugins/pbjx/components/history/index.jsx

```diff
@@ -3,8 +3,6 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
```

Comment:
@rsumilang several eslint warnings about props validation in this component

Reply:
![Screen Shot 2020-06-09 at 2 26 55 PM](https://user-images.githubusercontent.com/1748115/84202915-5ac8a200-aa5d-11ea-99f3-c968b325e06b.png)

I don't see any.

Reply:
<img width="892" alt="Screen Shot 2020-06-09 at 3 44 33 PM" src="https://user-images.githubusercontent.com/6315388/84208255-29090880-aa68-11ea-9dd6-2934f5c02899.png">